include/Gaffer/Path.h

```diff
@@ -39,6 +39,7 @@
 #define GAFFER_PATH_H
 
 #include "Gaffer/Export.h"
+#include "Gaffer/PathFilter.h"
```

Comment:
We're deliberately forward declaring PathFilter at L64, to avoid unnecessary includes like this one. Where this has caused problems before, it's either been that the include has been missing somewhere else (a cpp file where PathFilter is used), or that we've accidentally allowed the compiler to generate an inline function for us, and it has wanted the PathFilter definition for that. I remember one previous MSVC case where it was because we hadn't declared a destructor explicitly, for instance. Would be good to see if there's not an explanation like that in this case, before resorting to more includes.

If we do need the include, we should remove the forward declaration.

Reply:
The need for `PathFilter.h` was coming from using `FileSystemPath` to reformat paths to the generic format. I've removed those in 5bd9d5d8e541728e036bfae2d1827e1759961388 and 9df1ac0791b9b085dc06646b5847e62522fce485 so `PathFilter` isn't causing problems any more.